docs/developer/attributes/overview.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Attributes and types overview
```

Comment:
**style**: This title looks a bit big (maybe shorten it to "Attributes and Types"?).

**suggestion (blocking)**: Titles should adhere to [The Chicago Manual of Style](https://en.wikipedia.org/wiki/The_Chicago_Manual_of_Style).

```suggestion
title: Attributes and Types Overview
```